app/src/ui/lib/author-input/author-input.tsx

```diff
@@ -207,6 +211,7 @@ export class AuthorInput extends React.Component<
   private renderAuthors() {
     return (
       <div
+        id="added-authors"
         className="added-author-container"
         ref={this.authorContainerRef}
         aria-labelledby="author-input-label"
```

Comment:
Wouldn't having a `label` with `for` mean that this is redundant?

Reply:
You would think so, but VoiceOver didn't read it, and I couldn't set a `label` without its `for` or otherwise we get a linter error 😞

Reply:
🤦